v22.1/stream-a-changefeed-to-a-confluent-cloud-kafka-cluster.md

```diff
@@ -0,0 +1,267 @@
+## Step 4. Create a Confluent Schema Registry 
+
+In this step, you'll create the Schema Registry in your environment. 
+
+Enable the Schema Registry for the active environment: 
+
+~~~shell
+confluent schema-registry cluster enable --cloud "gcp" --geo "us"
+~~~
+
+The `--cloud` and `--geo` flags are **required** with this `enable` command. See the [`confluent schema-registry cluster enable`](https://docs.confluent.io/confluent-cli/current/command-reference/schema-registry/cluster/confluent_schema-registry_cluster_enable.html) docs for more detail. To match the Kafka cluster setup for this tutorial, the `cloud` is set to Google Cloud Platform and the `geo` to the US.
+
+You will receive output showing the Schema Registry's ID and its endpoint URL:
+
+~~~
++--------------+----------------------------------------------------+
+| Id           | lsrc-816zp7                                        |
+| Endpoint URL | https://psrc-x77pq.us-central1.gcp.confluent.cloud |
++--------------+----------------------------------------------------+
+~~~
+
+## Step 5. Create a Schema Registry API key and secret 
+
+Next, generate an API and secret key for the Schema Registry using the ID from your output:
+
+~~~shell 
+confluent api-key create --resource {SCHEMA REGISTRY ID}
+~~~
+
+The output will display your API key and secret. You'll need these to create your Kafka consumer and start your changefeed.
+
+## Step 6. Create a Kafka consumer 
+
+In this step, you'll start a Kafka consumer for the changefeed messages.
+
+Run the following command to create a consumer:
+
+~~~shell
+confluent kafka topic consume users \
+ --value-format avro \
+ --from-beginning \
+ --sr-endpoint https://{psrc-x77pq.us-central1.gcp}.confluent.cloud \
+ --sr-api-key {SCHEMA REGISTRY API Key} \
+ --sr-api-secret {SCHEMA REGISTRY SECRET}
+~~~
+
+In this command, you need to pass the following Schema Registry details:
+
+- The endpoint from the output in [Step 4](#step-4-create-a-confluent-schema-registry)
+- The API and secret key from [Step 5](#step-5-create-a-schema-registry-api-key-and-secret)
+
+Once you run this command, your terminal will wait for messages. 
+
+{{site.data.alerts.callout_success}}
+Run `confluent schema-registry cluster describe` to access details for the Schema Registry.
+{{site.data.alerts.end}}
+
+## Step 7. Create a changefeed 
+
+To create your changefeed, you'll prepare your CockroachDB cluster and the necessary authentication to your Kafka cluster and Schema Registry.
+
+In a new terminal window, start a SQL session for your CockroachDB cluster:
+
+~~~shell
+cockroach sql --url {CONNECTION STRING}
+~~~
+
+Before you can create an {{ site.data.products.enterprise }} changefeed, it is necessary to enable rangefeeds on your cluster:
+
+~~~sql 
+SET CLUSTER SETTING kv.rangefeed.enabled = true;
+~~~
+
+Before running the [`CREATE CHANGEFEED`](create-changefeed.html) statement, you must **URL-encode** both the cluster's and the Schema Registry's API secret key.
```

Comment:
Maybe we should link to a place that will encode for you? I always use this website: https://www.base64encoder.io/